Documentation CYF 1.0/pages/api-functions-waves.html

```diff
@@ -171,6 +171,15 @@ <h3>The Arena Object</h3><br><br>
                 <li><span class="CYF"></span> <span class="term">Arena.Show()</span> - Makes the Arena visible after using
                 <span class="term">Arena.Hide</span>.<br>
                 This is also called automatically whenever a wave ends!</li><br>
+
+                <li><span class="new066"></span> <span class="CYF"></span> <span class="term">Arena.SetSprite(<span class="string"> filename</span>)</span>
+                - Set Arena's sprite. The sprite is stretched to the size of the arena and is reset at the end of a wave.</li><br>
+
+                <li><span class="new066"></span> <span class="CYF"></span> <span class="term">Arena.SetColor(<span class="luatable"><span class="number"></span>, <span class="number"></span>, <span class="number"></span>, <span class="number"></span> = 1 </span>)</span>
```

Comment:
Also, I'm thinking maybe we should rename these to Arena.color and Arena.color32 to match CYF's other objects